tests/http/clients/base.py

```diff
@@ -122,8 +129,9 @@ def _build_body(
         variables: Optional[Dict[str, object]] = None,
         files: Optional[Dict[str, BytesIO]] = None,
         method: Literal["get", "post"] = "post",
+        extensions: Optional[Dict[str, Any]] = None,
     ) -> Optional[Dict[str, object]]:
-        if query is None:
+        if query is None and extensions is None:
             assert files is None
             assert variables is None
```

Comment:
is this additional check necessary? 🤔

Reply:
It's not. Was a part of the previous work I overseen